test/unit-test/test_lib/test_analysis/test_network.py

```diff
@@ -25,48 +25,51 @@ def tearDown(self):
         self.nick_same_list = None
 
 
-    @mock.patch('lib.analysis.network.config.MAX_EXPECTED_DIFF_NICKS', 5000)
-    @mock.patch('lib.analysis.network.config.THRESHOLD_MESSAGE_NUMBER_GRAPH', 0)
-    @mock.patch('lib.analysis.network.config.MINIMUM_NICK_LENGTH', 3)
-    @mock.patch('lib.analysis.network.config.DEBUGGER', True)
-    @mock.patch('lib.analysis.network.util', autospec=True)
-    def test_message_number_graph(self, mock_util):
+    @mock.patch('lib.config.MAX_EXPECTED_DIFF_NICKS', 5000)
+    @mock.patch('lib.config.THRESHOLD_MESSAGE_NUMBER_GRAPH', 0)
+    @mock.patch('lib.config.MINIMUM_NICK_LENGTH', 3)
+    @mock.patch('lib.config.DEBUGGER', True)
+    @mock.patch('lib.util.to_graph', autospec=True)
+    @mock.patch('lib.util.create_connected_nick_list', autospec=True)
+    @mock.patch('lib.util.check_if_msg_line', autospec=True)
+    @mock.patch('lib.util.correctLastCharCR', autospec=True)
+    @mock.patch('lib.util.rec_list_splice', autospec=True)
+    @mock.patch('lib.util.get_nick_sen_rec', autospec=True)
+    def test_message_number_graph(self, mock_get_nick_sen_rec, mock_rec_list_splice, mock_correctLastCharCR, \
+                                    mock_check_if_msg_line, mock_create_connected_nick_list, mock_to_graph):
         to_graph_ret = util.load_from_disk(current_directory + "/data/message_number_graph/to_graph")
         
         conn_list = list(connected_components(to_graph_ret))
         
-        mock_util.to_graph.return_value = to_graph_ret
-        mock_util.rec_list_splice.side_effect = util.load_from_disk(current_directory+ "/data/message_number_graph/rec_list_splice")
-        mock_util.create_connected_nick_list.return_value = util.load_from_disk(current_directory+ "/data/message_number_graph/conn_comp_list")
-        mock_util.check_if_msg_line.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/check_if_msg_line")
-        mock_util.correctLastCharCR.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/correctLastCharCR")
-        mock_util.get_nick_sen_rec.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/get_nick_sen_rec")
-        mock_util.extend_conversation_list.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/extend_conversation_list")
-        mock_util.get_nick_representative.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/get_nick_representative")
-        mock_util.correct_last_char_list.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/correct_last_char_list")
-        mock_util.get_year_month_day.side_effect = util.load_from_disk(current_directory + "/data/message_number_graph/get_year_month_day")
```

Comment:
This advice on not mocking the whole modules must be documented in unit test patterns.

Reply:
@prasadtalasila I have update the wiki.